main.py

```diff
@@ -50,18 +62,22 @@ def load_settings():
                     print(f"detected screen height {resolution_height}")
         png_path = f"listen-in-{resolution_height}.png"
         print(f"using png path {png_path}")
-    return png_path, confidence
+        pause_hotkey = settings["pause_hotkey"]
+    return png_path, confidence, pause_hotkey
 
 def do_loop(png_path, confidence):
     print("-------------------------------------")
     print("Trying to locate the listen in button")
     while True:
-        try:
-            time.sleep(1)
-            x, y = pyautogui.locateCenterOnScreen(png_path, confidence=confidence)
-            print(f"Clicking on listen in button on ({x}, {y})")
-            pyautogui.click(x, y)
-        except TypeError as e: pass
+        if not paused:
```

Comment:
this is a big neatpick, and you can just disregard this, but:

i would do 
```
if paused:
  continue
```

Doing the code flow this way will reduce the nesting and make the code more readable